contracts/random/README.md

```diff
@@ -0,0 +1,62 @@
+# Random Number Generation
+
+This directory contains contracts that provide random number generation capability using on-chain and off-chain sources. 
+
+The reasons for using these contracts are that:
+
+* Enables you to leverage a random number generation system designed by Immutable's cryptographers.
+* Allows you to build your game against an API that won't change.
+* The quality of the random numbers generated will improve as new capabilities are added to the platform. That is, the migration from ```block.hash``` to ```block.prevrandao``` when the BFT fork occurs will be seemless.
+* For off-chain randomness, allows you to leverage the random number provider that Immutable has agreements with.
+
+# Status
+
+Contract audits:
+
+| Description               | Date             |Version Audited  | Link to Report |
+|---------------------------|------------------|-----------------|----------------|
+| Not audited               | -                | -               | -              |
+
+Deployments:
+
+| Location                  | Version Deployed | Address |
+|---------------------------|------------------|---------|
+| Immutable zkEVM Testnet   | Not deployed     | -       |
+| Immutable zkEVM Mainnet   | Not deployed     | -       |
+
+## Architecture
+
+The Random Number Generation system on the immutable platform is shown in the diagram below.
+
+![Random number genration](./random-architecture.png)
+
+Game contracts extend ```RandomValues.sol```. This contract interacts with the ```RandomSeedProvider.sol``` contract to request and retreive random seed values. 
+
+There is one ```RandomSeedProvider.sol``` contract deployed per chain. Each game has its own instance of ```RandomValues.sol``` as this contract is integrated directly into the game contract. 
```

Comment:
nit: out of curiosity, is there a reason you use triple back ticks instead of a single back ticks for file names and identifiers here? Given they are not code blocks, I would have though a single back tick should be fine

Reply:
I have always used triple ticks. I didn't know single ticks would work.